Example/Tests/HaventecAuthenticateTests.swift

```diff
@@ -80,6 +80,29 @@ class HaventecAuthenticateTest: XCTestCase {
         XCTAssertThrowsError(try HaventecAuthenticate.getUserUuid())
     }
 
+    func testEdgeCases() throws {
```

Comment:
I don't know what the edge case is, break down into specific test methods

Reply:
Multiple edge cases. The asserts are readable